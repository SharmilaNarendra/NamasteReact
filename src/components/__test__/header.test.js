import { render } from "@testing-library/react";
import Header from "../header/HeaderComponent";

test("test loading header", () => {
  render(<Header />);
});
