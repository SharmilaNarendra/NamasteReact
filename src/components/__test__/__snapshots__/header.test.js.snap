// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test loading header 1`] = `
{
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="bg-purple-900 header text-white text-7xl fixed top-0 left-0 w-full h-24 z-1000 flex flex-row gap-7 dark:bg-slate-800"
      >
        <img
          alt=""
          data-testid="logo"
          src="test-file-stub"
        />
        <div
          class="self-center basis-5"
        >
          HUSTLERS
        </div>
        <img
          alt=""
          class="h-12 w-12 self-center cursor-pointer ml-[60%]"
          src="test-file-stub"
        />
        <img
          alt=""
          class="user-icon h-14 w-14 self-center cursor-pointer ml-[1%]"
          src="test-file-stub"
        />
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="bg-purple-900 header text-white text-7xl fixed top-0 left-0 w-full h-24 z-1000 flex flex-row gap-7 dark:bg-slate-800"
    >
      <img
        alt=""
        data-testid="logo"
        src="test-file-stub"
      />
      <div
        class="self-center basis-5"
      >
        HUSTLERS
      </div>
      <img
        alt=""
        class="h-12 w-12 self-center cursor-pointer ml-[60%]"
        src="test-file-stub"
      />
      <img
        alt=""
        class="user-icon h-14 w-14 self-center cursor-pointer ml-[1%]"
        src="test-file-stub"
      />
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
